src/lint/mod.rs

```diff
@@ -60,10 +59,13 @@ pub struct LintCfg {
     pub continuation_line: Option<ContinuationLineOptions>,
     #[serde(default)]
     pub in9: Option<IN9Options>,
+    #[serde(default)]
+    pub in1_common_indentation: Option<u32>,
 }
 
 impl Default for LintCfg {
     fn default() -> LintCfg {
+        let in1_common_indentation = Some(4);
```

Comment:
Don't remove `INDENTATION_LEVEL_DEFAULT` as it would be better used here instead of the hardcoded 4 value.

Anyways, I think this default method could work from the options of in1 also, avoiding this let statement.